datacommons_pandas/CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+
+## 0.0.1
+
+**Date** - 08/24/2020
+
+**Release Tag** - [pd.0.0.1](https://github.com/datacommonsorg/api-python/releases/tag/pd0.0.1)
```

Comment:
question: do we always release two packages in the same time? or they can be released separately?

Reply:
They can be released separately